### PR TITLE
Add viewport

### PR DESCRIPTION
**Pull request by R4356th**
### Changes

Adds viewport.

### Cause of Changes

This single line should be able to make mobile and tablet experience better. You can read how that works [here](https://developer.mozilla.org/en-US/docs/Mozilla/Mobile/Viewport_meta_tag).

### Extra Note

GitHub is showing the diff incorrectly; the indentation is alright.

### CODE REVIEW

**Comment by jmoenig** [comment]
@bromagosa this is for you to review ;-)

File: snap.html [hunk 1]
```diff
@@ -2,6 +2,7 @@
 <html>
     <head>
         <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
```
Comment (towerofnix) [inline snap.html:5]: It's not a huge deal, but (almost) the rest of this file uses spaces and this is a tab, so maybe you could replace the tab with eight manually-typed spaces (or switch your editor to use spaces instead of tabs)?

Comment (R4356th) [reply snap.html:5]: I do not think this is an issue because it looks alright when you view the file.

Comment (R4356th) [reply snap.html]: Never mind. I just fixed this while resolving the conflict.

**Comment by bromagosa** [comment]
@jmoenig this looks good to me. Just tested it in Android and it does make it significantly more usable. Prevents the page from automatically zooming all over the place :)

A phone is still not the ideal environment for Snap!, but an Android tablet is actually not bad at all!

We should find a way to automatically hide the virtual keyboard after the user's finished typing, but that's a different topic.

TLDR; this is a simple change that significantly improves usability in Android. Should be pulled :)

**Comment by jmoenig** [comment]
Thanks for the test and review, @bromagosa, and thanks, @R4356th for the PR. Will pull for the next patch.
